package.json

```
@@ -1,14 +1,20 @@
 {
   "name": "moment-relative",
-  "version": "1.1.0",
+  "version": "2.0.0",
   "description": "Support for graphite-style relative times in Moment",
   "main": "index.js",
   "repository": {
     "type": "git",
     "url": "git://github.com/bbc/moment-relative.git"
   },
+  "engines": {
+    "node": ">4.0.0"
```

Think you meant this:

```suggestion
    "node": ">=4.0.0"
```

---

Indeed. Let's move to 6.